web/webkit/src/main/scala/net/liftweb/http/LiftSession.scala

```diff
@@ -2345,15 +2345,31 @@ class LiftSession(private[http] val _contextPath: String, val underlyingId: Stri
    * the CometActor is instantiated.  If the CometActor already exists
    * in the session, the message will be sent immediately.  If the CometActor
    * is not yet instantiated, the message will be sent to the CometActor
-   * as part of setup (@see queueCometMessage) regardless of if it is created as part
+   * as part of setup (see queueCometMessage) if it is created as part
+   * of the current HTTP request/response cycle.
+   *
+   * @param msg the message to send to the CometActor
+   */
+  def sendCometMessage[T](msg: Any)(implicit cometManifest: Manifest[T]): Unit = {
+    val castClass = cometManifest.runtimeClass.asInstanceOf[Class[T]]
+    val typeName = castClass.getSimpleName
+    sendCometMessage(typeName, msg)
+  }
+
+  /**
+   * This method will send a message to a CometActor, whether or not
+   * the CometActor is instantiated.  If the CometActor already exists
+   * in the session, the message will be sent immediately.  If the CometActor
+   * is not yet instantiated, the message will be sent to the CometActor
+   * as part of setup (see queueCometMessage) regardless of if it is created as part
    * of the current HTTP request/response cycle. We plan to restrict this to
    * the current request/response cycle in the future (as that is the intended beavior).
    *
    * @param theType the type of the CometActor
    * @param name the optional name of the CometActor
    * @param msg the message to send to the CometActor
    */
-  def sendCometActorMessage(theType: String, name: Box[String], msg: Any) {
+  def sendCometMessage(theType: String, name: Box[String], msg: Any): Unit = {
```

Comment:
I thought it more important to make the scaladocs reference the common overload (imagine that the central implementation is done one way and changed to work a different way; the likelihood someone updates the documentation on all the callers is near 0) than to actually move the code around, honestly :)